treecraft/core/file_handlers.py

```
@@ -0,0 +1,399 @@
+
+def fix_malformed_fasta(file_path):
+    """
+    Fix a malformed FASTA file by splitting lines that contain embedded headers.
+    
+    Args:
+        file_path: Path to the malformed FASTA file
+        
+    Returns:
+        List of Bio.SeqRecord objects from the fixed file
+    """
+    from io import StringIO
```

Ruff (0.8.2)

`io.StringIO` imported but unused

Remove unused import: `io.StringIO`

(F401)